src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -102,11 +102,14 @@ var fluid = fluid || fluid_2_0_0;
         invertConfiguration: "fluid.transforms.invertToIdentity"
     });
 
-    fluid.transforms.round = function (value) {
-        return Math.round(value);
+    fluid.transforms.round = function (value, transformSpec) {
+        if (typeof transformSpec.scale === "number" && !isNaN(transformSpec.scale)) {
```

Comment:
@amb26 and I discussed this in the [fluid-work channel](https://botbot.me/freenode/fluid-work/2017-07-27/?msg=89126963&page=1). We decided to use fluid.roundToDecimal for all the rounding.